### DISCUSSION TITLE
Plot random orientations out of given orientations

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#1896] [Ask Anything] [answered]
Hi,
I have a .txt file containing 1000 orientations, and when I plot a pole figure based on these orientations. 
MTEX says: I'm plotting 208 random orientations out of 1000 given orientations
What does it mean? Why are there an additional 208 orientations？
Thank you so much.

**Comment by lfgmorales** [comment, marked as answer]
MTEX plots only 208 orientations out of 1000. if you want to specify the number of points, you have to give the ‚points‘,

Cheers

Luiz



Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 11 Jul 2023, at 11:13, louis96108 ***@***.***> wrote:

﻿

Hi,
I have a .txt file containing 1000 orientations, and when I plot a pole figure based on these orientations.
MTEX says: I'm plotting 208 random orientations out of 1000 given orientations
What does it mean? Why are there an additional 208 orientations？
Thank you so much.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1896>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZ3KQ7MJ2YULSKN7U3XPUKLPANCNFSM6AAAAAA2FWGXXY>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1896", "url": "https://github.com/mtex-toolbox/mtex/discussions/1896", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
